public/js/graph/setup.js

```diff
@@ -2,13 +2,17 @@
 // Globals
 var timetable = 'static/res/timetable.html';
 var clickedCourses = [];
-var FCEs = 0;
+
+// FCEs for current graph
+var FCEs = 0; 
```

Comment:
I see. If `FCEs` tracks the current FCEs selected on the current department, then why don't you:

1) rename the variables
2) display its value alongside the total FCE count (perhaps just in parentheses after the global total, for now)